src/main/views/includes/scripts.njk

```diff
@@ -7,6 +7,7 @@
 <script src="{{ asset_paths['js'] }}/detectJavascript.js"></script>
 <script src="{{ asset_paths['js'] }}/analytics.js"></script>
 <script src="{{ asset_paths['js'] }}/analytics-custom-event-sender.js?2017-10-11"></script>
+<script src="{{ asset_paths['js'] }}/double-click-preventer.js"></script>
```

Comment:
when you add querySelectorAll you should add above it the nodelist polyfill,
`<script src="{{ asset_paths['js_vendor'] }}/nodelist-foreach-polyfill.js"></script>`